http/headers/Link.json

```diff
@@ -34,6 +34,119 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "blocking": {
+          "__compat": {
+            "spec_url": "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-blocking",
+            "tags": [
+              "web-features:blocking-render"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "105"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1751383"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false,
+                "impl_url": "https://webkit.org/b/267232"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
```

Comment:
✅ This is the same as:

https://github.com/mdn/browser-compat-data/blob/6295ecf5d57dc8dd197ab3b423b76dca2555768c/api/HTMLLinkElement.json#L88-L127